src/utils/gauges.ts

```diff
@@ -232,8 +238,9 @@ export async function getGaugeData(
           gaugeName: gaugeNames[index],
           lpTokenAddress,
           isKilled: gaugeKillStatuses[index] ?? false,
-          poolAddress: isValidPoolAddress && poolAddress ? poolAddress : null,
-          poolName: gaugePool?.poolName || null,
+          poolAddress:
+            isValidPoolAddress && pool?.poolAddress ? pool?.poolAddress : null,
```

Comment:
does it make sense to early return here? you should also be able to do `poolAddress: isValidPoolAddress && pool?.poolAddress` to shorten logic

Reply:
agreed